### ISSUE TITLE
Spaces in folder names don't work right

### ISSUE THREAD

**Issue by twist84** [rexdex/recompiler#23]
If you have a game in `D:\Games\Consoles\X360\Homebrew\Quake 3\` after `Quake ` the rest of the folder name doesn't get parsed.

**Comment by rexdex** [comment]
Is this in the GUI tools or via command line ?

**Comment by twist84** [comment]
GUI tools, I haven't checked via command line

**Comment by twist84** [comment]
Just checked via command line and it's the same even with quotation around the path`"D:\Games\Consoles\X360\Homebrew\Quake 3\q3_360.xex"`

**Comment by sumit0190** [comment]
Yeah, that looks about right. `CommandLine::Parse` has this loop where we get the value of each property specified, and we do this by moving forward on the string, one character at a time, until we encounter a space. It's worth noting that `argv` actually gives us the whole path (with spaces, all in quotes), but we screw up when parsing it.

@rexdex I can take this if you want. Not sure if the GUI tools use the same API.

**Comment by rexdex** [comment]
Yeah, go ahead. In general the space is a separator unless it's in the "", maybe the logic is flawed there.

**Comment by abaska** [comment]
Jesus i was learning c++ for two months to write an emulator like you two rex and sumit but it looks like i need years to reach an acceptable level of experience :(
